app/CommandExecutor.php

```diff
@@ -83,19 +105,62 @@ public function execute(string $command, string $cwd) : int
         return proc_close($proc);
     }
 
+    public function getOutputBuffer(): string
+    {
+        return $this->getFileContents($this->outputFileName);
+    }
+
+    public function getErrorBuffer(): string
+    {
+        return $this->getFileContents($this->errorFileName);
+    }
+
+    private function getFileContents(string $filename): string
+    {
+        $output = '';
+        $handle = @fopen($filename, 'r');
+
+        while (($buffer = fgets($handle)) !== false) {
+            $output .= trim($buffer) . PHP_EOL;
+        }
+
+        if (! feof($handle)) {
+            $output = 'Erro: falha inexperada na leitura do arquivo!';
+        }
+
+        return $output;
+    }
+
     protected function getDescriptors() : array
     {
         if ($this->output || env('FWD_VERBOSE')) {
             return [STDIN, STDOUT, STDERR];
         }
 
-        $devNull = fopen('/dev/null', 'w');
-
-        return [STDIN, $devNull, $devNull];
+        $this->outputFileName = @tempnam(sys_get_temp_dir(), 'fwd_output_');
```

Comment:
also, i would do this on `__construct`, as you already have the `destruct` method